cocoa/tests_backend/window.py

```diff
@@ -34,7 +34,7 @@ async def wait_for_window(
             delay=(
                 1.75
                 if state_switch_not_from_normal
-                else 0.75 if full_screen else 0.5 if minimize else 0.1
+                else 1 if full_screen else 0.5 if minimize else 0.1
```

Comment:
I'm unclear how this change would impact the failure mode that has been reported. The test failure is on L812; the preceding `wait_for_window` call explicitly sets `state_switch_not_from_normal=True`. So... shouldn't this be a change to the 1.75, not the line modified here?

Reply:
My running theory about this error is that the window cleanup state change isn't getting enough delay in case of fullscreen when it finishes running the first test case.
https://github.com/beeware/toga/blob/df65161f19fe131ef913eb473bca8248531537c4/testbed/tests/window/test_window.py#L764-L769
https://github.com/beeware/toga/blob/df65161f19fe131ef913eb473bca8248531537c4/testbed/tests/conftest.py#L87-L114

I tried to increase `state_switch_not_from_normal` delay to 3, but it still fails. So, I am going to try 2 solutions:
* Add a waiting delay to the start of the test
* Manually reset the window state to NORMAL at the end of the test